src/stringify.ts

```diff
@@ -17,19 +17,22 @@ import SessionData from './types/SessionData';
 import Variant from './types/Variant';
 import MediaInitializationSection from './types/MediaInitializationSection';
 
-const ALLOW_REDUNDANCY = [
+const ALLOW_REDUNDANCY: Set<string> = new Set([
     '#EXTINF',
     '#EXT-X-BYTERANGE',
     '#EXT-X-DISCONTINUITY',
     '#EXT-X-STREAM-INF',
+    '#EXT-X-CUE-OUT-CONT',
     '#EXT-X-CUE-OUT',
     '#EXT-X-CUE-IN',
     '#EXT-X-TRANSMIT-CUE-OUT',
     '#EXT-X-KEY',
     '#EXT-X-MAP',
-] as const;
+]);
 
-const SKIP_IF_REDUNDANT = ['#EXT-X-MEDIA'] as const;
+const TAG_REGEX = /^(?<tag>#EXT[A-Z-]+).*$/;
```

Comment:
I think you could go event easier and just check that the first index of the string is `#`. Flipping the logic also allows you to identify URIs.